2022-05-29 17:44:00,007 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2022-05-29 17:44:00,007 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\Python39\lib\site-packages\instabot\bot\bot.py
2022-05-29 17:44:00,008 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2022-05-29 17:44:00,285 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2022-05-29 17:44:00,285 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2022-05-29 17:44:00,286 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2022-05-29 17:44:00,587 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2022-05-29 17:44:00,985 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2022-05-29 17:44:01,150 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [429]>
2022-05-29 17:44:01,150 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2022-05-29 17:44:01,151 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2022-05-29 17:44:03,998 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [400]>
2022-05-29 17:44:03,998 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https://i.instagram.com/challenge/AXFma9Xrx3NCk14A35JQlcWIMm5_o8KjkPwHJbP7qaYxfL9m0jDCZ-qFtFM81BRhcFsPcpQ/nD0wkuqQz6FbR4WA/","api_path":"/challenge/AXFma9Xrx3NCk14A35JQlcWIMm5_o8KjkPwHJbP7qaYxfL9m0jDCZ-qFtFM81BRhcFsPcpQ/nD0wkuqQz6FbR4WA/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail","error_type":"checkpoint_challenge_required"}'
2022-05-29 17:44:03,998 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2022-05-29 17:44:03,998 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2022-05-29 17:44:03,998 - instabot version: 0.117.0 (api) - INFO - Error type: checkpoint_challenge_required
